src/locales/ja.json

```diff
@@ -0,0 +1,71 @@
+{
+ "All items were canceled from the order": "全てのアイテムが注文からキャンセルされました",
```

Comment:
Hi,
Please convert the identation to 1 tab (2 space).
```suggestion
  "All items were canceled from the order": "全てのアイテムが注文からキャンセルされました",
```

Reply:
@ymaheshwari1 
revised it. please check it.
thank you.

Reply:
Looks good. Thanks for the contribution